unstructured/ingest/connector/fsspec.py

```diff
@@ -111,13 +127,76 @@ def get_file(self):
             **self.config.get_access_kwargs(),
         )
         logger.debug(f"Fetching {self} - PID: {os.getpid()}")
-        fs.get(rpath=self.remote_file_path, lpath=self._tmp_download_file().as_posix())
+        try:
+            fs.get(rpath=self.remote_file_path, lpath=self._tmp_download_file().as_posix())
+        except FileNotFoundError as e:
+            logger.error("Failed to find file in remote source.")
+            logger.error(e)
+
+    def _catch_not_implemented(self, method, **kwargs) -> Optional[str]:
```

Comment:
this name is confusing. really what we're doing here is attempting to fetch a date for a file path using an arbitrary method. it happens to account for NotImplementedError. call this something like `_get_formatted_date`?

Reply:
The NotImplementedError catching is secondary to that imo.

Reply:
the only method that doesn't return a date that we're passing through this method (incorrectly) is the fs.sign. I think in that case we should just put the try/catch inline rather than going through another function